profiler/src/ProfilerEngine/Datadog.Profiler.Native/RuntimeIdStore.cpp

```diff
@@ -0,0 +1,203 @@
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2022 Datadog, Inc.
+
+#include "RuntimeIdStore.h"
+
+#include "Log.h"
+#include "shared/src/native-src/string.h"
+
+#ifdef _WINDOWS
+#define LIBRARY_FILE_EXTENSION ".dll"
+#elif LINUX
+#define LIBRARY_FILE_EXTENSION ".so"
+#elif MACOS
+#define LIBRARY_FILE_EXTENSION ".dylib"
+#else
+Error("unknown platform");
+#endif
+
+const char* const RuntimeIdStore::ServiceName = "RuntimeID Store";
+const char* const RuntimeIdStore::ExternalFunctionName = "GetRuntimeId";
+const char* const RuntimeIdStore::NativeLoaderFilename =
+#ifdef BIT64
+    "Datadog.AutoInstrumentation.NativeLoader.x64" LIBRARY_FILE_EXTENSION;
+#else
+    "Datadog.AutoInstrumentation.NativeLoader.x86" LIBRARY_FILE_EXTENSION;
+#endif
+
+extern "C"
+{
+#ifdef WIN32
```

Comment:
we should be consistent and use _WINDOWS everywhere instead of WIN32 and _WIN32 (applies to the entire file)

Reply:
✔️